endpoints/citrus-camel/src/main/java/com/consol/citrus/camel/dsl/CamelSupport.java

```diff
@@ -109,8 +111,8 @@ public CamelMessageProcessor.Builder process(Processor processor) {
      * @return
      */
     public CamelRouteActionBuilder route() {
-        return new CamelRouteActionBuilder()
-                .context(camelContext);
+        return new CamelActionBuilder()
+                .camelContext(camelContext).route();
```

Comment:
to be uniform with line breaks.

```suggestion
                .camelContext(camelContext)
                .route();
```

Reply:
👍